File: exs/mundo_2/ruby/038.rb
=begin
Desafio 038

Problema: Escreva um programa que leia dois números inteiros e compare-os.
          mostrando na tela uma mensagem:
            - O primeiro valor é maior
            - O segundo valor é maior
            - Não existe valor maior, os dois são iguais

Resolução do problema:
=end

print"Digite o primeiro número: "
n1 = gets.chomp.to_i

print"Digite o segundo número: "
n2 = gets.chomp.to_i

if n1 > n2
	puts"#{n1} é maior que #{n2}."
elsif n1 < n2
	puts"#{n2} é maior que #{n1}."
else
	puts"Os dois números são iguais."
end

File: exs/mundo_2/java/038.java
/*
Desafio 038

Problema: Escreva um programa que leia dois números inteiros e compare-os.
          mostrando na tela uma mensagem:
            - O primeiro valor é maior
            - O segundo valor é maior
            - Não existe valor maior, os dois são iguais

Resolução do problema:
*/

import java.util.Scanner;

class Main{
    public static void main(String[] args) {

        // instancia a classe Scanner
        Scanner scanner = new Scanner(System.in);

        System.out.println("Digite um numero abaixo: ");
        double numero1 = scanner.nextInt();

        System.out.println("Digite outro número:");
        double numero2 = scanner.nextInt();

        if (numero1 > numero2) {
            System.out.println("O primeiro número é maior");
        } else if (numero1 < numero2) {
            System.out.println("O segundo número é maior");
        } else System.out.println("Não existe maior, os dois são IGUAIS!");

    }
}